run_dir/static/css/status.css

```diff
@@ -26,35 +26,19 @@ multiple places and typically control typography, or other common elements.
 
 
 /* index.html page */
-#project_quotas .plot-column {
-	height: 60px;
-	padding: 0;
-	overflow: hidden;
-}
-
-#project_quotas tr td:first-child {
-	width: 150px;
-}
-
-.update_table {
-	overflow: hidden;
-}
-
-#project_quotas td {
+.table-disk-quotas > tbody > tr > td {
   padding: 0;
 }
-#project_quotas td:first-child {
```

Comment:
You still need this chunk, otherwise the first table cell looks weird:

```css
.table-disk-quotas > tbody > tr > td:first-child {
  padding: 8px;
}
```

Without it you get this:
![image](https://cloud.githubusercontent.com/assets/465550/13113353/f5118442-d58e-11e5-9a8d-155083f759bb.png)


instead of this:
![image](https://cloud.githubusercontent.com/assets/465550/13113360/fc3bcd9a-d58e-11e5-98bb-8e08f4c298be.png)